external/model-preparation-algorithm/configs/classification/configuration.yaml

```diff
@@ -123,15 +123,32 @@ learning_parameters:
     default_value: true
     description: Adaptive early exit from training when accuracy isn't changed or decreased for several epochs.
     editable: true
-    header: Enable adaptive early stopping of the training
+    header: Enable early stopping of the training
     type: BOOLEAN
     ui_rules:
       action: DISABLE_EDITING
       operator: AND
       rules: []
       type: UI_RULES
-    visible_in_ui: false
+    visible_in_ui: true
     warning: null
+  patience:
+    affects_outcome_of: TRAINING
+    default_value: 10
+    description: Number of epochs without improvement before the system will automatically stop training.
```

Comment:
How about "Training will be stopped if the model is not improved during patience."